code/_onclick/cyborg.dm

```diff
@@ -105,10 +105,26 @@
 // for non-doors/apcs
 /mob/living/silicon/robot/CtrlShiftClickOn(atom/A)
 	A.BorgCtrlShiftClick(src)
+	var/obj/machinery/door/airlock/airlock = locate(/obj/machinery/door/airlock) in A // Skyrat edit
+	if(airlock)
+		airlock.AICtrlShiftClick(src)
+	return // End of skyrat edit
+
 /mob/living/silicon/robot/ShiftClickOn(atom/A)
 	A.BorgShiftClick(src)
+	A.AIShiftClick(src)
```

Comment:
```suggestion
```

Reply:
Removed that , pasted it there by accident.